apps/web/__tests__/test/smoke/cartPage.cy.ts

```diff
@@ -1,28 +1,25 @@
 import { CartPageObject } from '../../support/pageObjects/CartPageObject';
-import { paths } from '../../../utils/paths';
 
 const cart = new CartPageObject();
 
 describe('Smoke: Cart Page', () => {
-  beforeEach(() => {
-    cy.setCookie('vsf-locale', 'en');
-    cy.setCookie(
-      'consent-cookie',
-      '{"Essentials":{"Session":true,"Consent":true,"Session2":true},"External Media":{"Session":false,"Consent":false,"Session2":false},"Functional":{"Session":false,"Consent":false,"Session2":false},"Marketing":{"Session":false,"Consent":false,"Session2":false}}',
-    );
-  });
-
   it('[smoke] Add coupon to cart, check the totals, then remove coupon and check totals again', () => {
     cy.visitAndHydrate('/study-room-office/office-chair/design-chair-brookhaven-leather-black_105_1003');
-    cy.intercept('/plentysystems/doAddCartItem').as('doAddCartItem');
-    cy.wait(1000);
-    cy.getByTestId('add-to-cart').click();
-    cy.wait('@doAddCartItem');
-    cy.visitAndHydrate(paths.cart);
-    cart.openCouponAccordion();
-    cart.addCoupon('KB82AZ');
-    cart.orderSummayAfterCouponApplyed('-£12.91', '£119.99');
-    cart.removeCoupon();
-    cart.orderSummayAfterCouponRemoved('£132.90');
+
+    cy.wait(1000)
+      .intercept('/plentysystems/doAddCartItem')
+      .as('doAddCartItem')
+      .wait(1000)
```

Comment:
**question**: Are you sure both `wait(1000)` statements are necessary? Cypress should retry for a few seconds by itself.